tests/unit/s2n_security_policies_test.c

```diff
@@ -49,6 +50,39 @@ int main(int argc, char **argv)
                 EXPECT_TRUE(kem_group_is_supported);
             }
         }
+
+        /* TLS 1.3 Cipher suites have TLS 1.3 Signature Algorithms Test */
+        int has_tls_13_cipher = 0;
+        for(int i = 0; i < security_policy->cipher_preferences->count; i++){
+            if (security_policy->cipher_preferences->suites[i]->minimum_required_tls_version == S2N_TLS13) {
+                has_tls_13_cipher = 1;
+                break;
+            }
+        }
+
+        /* If a TLS 1.3 Cipher is present in the Security Policy, then the minimum required TLS 1.3 signature algorithms
+         * must be present as well. */
+        if (has_tls_13_cipher) {
+            int has_tls_13_sig_alg = 0;
+            int has_rsa_pss = 0;
+
+            for(int i = 0; i < security_policy->signature_preferences->count; i++) {
+                int min = security_policy->signature_preferences->signature_schemes[i]->minimum_protocol_version;
+                int max = security_policy->signature_preferences->signature_schemes[i]->maximum_protocol_version;
+                s2n_signature_algorithm sig_alg = security_policy->signature_preferences->signature_schemes[i]->sig_alg;
+
+                if (min >= S2N_TLS13 || max <= S2N_TLS13) {
```

Comment:
I meant to check if either is greater-than-or-equal. Fixed. I also need to check both min and max since some are not populated: https://github.com/aws/s2n-tls/blob/main/tls/s2n_signature_scheme.c#L25-L210